### ISSUE TITLE
For all components, support `high-contrast` mode

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1362]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### Device OS version

26

### OUDS iOS library version

1.3.0

### Describe the issue

As done for radio and checkbox, chip should support High-contrast mode

### Expected Behavior

Support high-contrast mode for each componen, including Chip

### Accessibility settings

high-contrast mode

**Comment by pylapp** [comment]
@MaximeTonnerre @TonyAntonyHello @jerome-regnier @HamzaAmarirHA  Quelles seraient les règles à appliquer pour la gestion du mode contrast élevé sur les composats ?
Je pense notamment aux couleurs de bordures, de _foreground_ et de _backgroud_.

**Comment by paulinea** [comment]
@pylapp @ludovic35 

Voici les règles appliquées sur Android et validées.

La gestion du contraste élevé est disponible pour les composants Checkbox, Radio Button et Filter Chip. Lorsque l'utilisateur a activé le high contrast sur son terminal, on modifie les couleurs de ces composants uniquement quand ils sont sélectionnés.
On applique le token `color/always/black` aux couleurs foncées et `color/always/white` aux couleurs claires.

Voici les rendus validés côté Android :

<img width="159" height="206" alt="Image" src="https://github.com/user-attachments/assets/0f6e9d73-3477-4b3a-93f1-33662eb438bc" />
<img width="165" height="207" alt="Image" src="https://github.com/user-attachments/assets/2650db51-51c5-42af-b3bd-8af98a155610" />
<img width="170" height="208" alt="Image" src="https://github.com/user-attachments/assets/ed3f0711-b93d-4f8d-a773-8e11b73ce12d" />
<img width="310" height="209" alt="Image" src="https://github.com/user-attachments/assets/5214c3c7-4590-45c9-a044-11955b0d0e74" />
<img width="316" height="204" alt="Image" src="https://github.com/user-attachments/assets/a96d6037-301b-4528-85b0-d502ddc3a95e" />
<img width="339" height="214" alt="Image" src="https://github.com/user-attachments/assets/b34bcd03-8bc9-4893-8545-1d13135c565e" />
